src/impl/conv_im2col.jl

```diff
@@ -51,7 +51,7 @@ function conv_im2col!(
         im2col!(col_slice, view(x, :, :, :, :, batch_idx), cdims)
         GC.@preserve col_slice w y begin
             col_ptr = pointer(col_slice)
-            w_ptr = pointer(w)
+            w_ptr = pointer(copy(conj(w)))
             y_ptr = pointer(y, (batch_idx - 1)*M*N + 1)
             gemm!(Val(false), Val(false), M, N, K, alpha, col_ptr, w_ptr, beta, y_ptr)
```

Comment:
This change was a little awkward for me.
All dense convolution tests are green, if I simply change the last line here:
```julia
w_ptr = pointer(w)
y_ptr = pointer(y, (batch_idx - 1)*M*N + 1)
gemm!(Val(false), Val(true), M, N, K, alpha, col_ptr, w_ptr, beta, y_ptr)
```
Note the `Val(true)` here. I wondered, if `Val(false)` is actually correct?
The `gradtest`s are failing with this change, though. So I left it as above.

Reply:
Interesting, looking at CI I wonder if this fixed the flaky gradtests we have when nthreads > 1!

Reply:
No, what I meant to say is that with
```julia
gemm!(Val(false), Val(true), M, N, K, alpha, col_ptr, w_ptr, beta, y_ptr)
```
all gradtests are failing.

Reply:
Yes, sorry. I wasn't referring to the change you wrote, but to the diff here with `copy`.